examples/transfer-jetton.ts

```diff
@@ -0,0 +1,24 @@
+import { Address, toNano } from "@ton/core";
+import { GameFiSDK, createWalletV4 } from "../src/index";
+
+async function main() {
+    const sdk = await GameFiSDK.create({
+        storage: {
+            pinataApiKey: process.env.PINATA_API!,
+            pinataSecretKey: process.env.PINATA_SECRET!,
+        },
+        api: 'testnet',
+        wallet: await createWalletV4(process.env.MNEMONIC!),
+    });
+
+    console.log('Using wallet', sdk.sender?.address);
+
+    const jetton = await sdk.openJetton(Address.parse("my-jetton-address"));
+
+    await (await jetton.getWallet(Address.parse('alice-address'))).sendTransfer({
```

Comment:
Lets extract `await jetton.getWallet(...)` as a separate variable `myWallet`, also please use `sdk.sender!.address` instead of some random (`alice-address`) address, since we can only control our own wallet